backend/src/main/java/com/ody/route/service/RouteService.java

```diff
@@ -21,7 +21,7 @@ public RouteTime calculateRouteTime(Coordinates origin, Coordinates target) {
             try {
                 RouteTime routeTime = client.calculateRouteTime(origin, target);
                 apiCallService.increaseCountByRouteClient(client);
-                log.info("{}를 사용한 소요 시간 계산 성공", client.getClass().getSimpleName());
+                log.info("{}를 사용한 소요 시간 계산 성공 - {}분", client.getClass().getSimpleName(), routeTime.getMinutes());
```

Comment:
좋은 로깅이라 생각해요. 그런데 두가지 의견 남기면

**첫째로, 현재 branch에 구현이 되어 있는지 모르겠지만 getClientType을 활용해도 좋을 것 같아요**. client.getClass().getSimpleName() 보다 type을 기록한다는 점이 더 명확할 것 같아요.

둘째로, 저번에 해음이랑 로그 트래킹을 하면서 느낀 점은 단순 호출 성공 결과가 중요한 것이 아니라, **누구의 시간을 호출했는지 트래킹하는 것이 중요**하더라고요. 즉 어떤 요청에서 누구의 호출을 했는지가 굉장히 중요했어요.

MDC를 활용해서 service 단에서 mateId를 넣어주고
MDC.put("requestMateId", requestMateId)
```suggestion
                log.info("mateId : {}, {}를 사용한 소요 시간 계산 성공 - {}분",  MDC.get("mateId", )client.getClass().getSimpleName(), routeTime.getMinutes());
```
현재 로깅 단계에서 mateID를 꺼내어 같이 로깅해주면 어떤 mate의 시간 계산인지가 명화갛게 드러나서 좋을 것 같아요.

MDC 필터에서 각 스레드별 clear 작업을 하고 있으니 필요한 값을 넣고 빼는 작업에 대한 후속처리는 크게 신경쓰지 않아도 될 것 같아요

Reply:
> getClientType을 활용

반영 완료!

> 누구의 시간을 호출했는지 트래킹

1. eta API 에서는 route time 계산 전 mateId가 있기 때문에 로깅이 가능한데,
2. save mate API에서는 계산 후 정보를 바탕으로 mate를 생성하고 있어, mateId 포함하여 로깅이 불가능해요
아래 처럼 2번 경우엔 `mateId=null` 로 표시됩니다!
```
2024-10-23 13:05:16.464 [INFO] [http-nio-8080-exec-2] [f44d0bac-6141-4ae9-b74d-49e0b7272b1c] [com.ody.route.service.RouteService] - mateId : null - GOOGLE API 사용한 소요 시간 계산 : 19분
```